cypress/e2e/registration_validation.spec.ts

```diff
@@ -322,10 +322,6 @@ describe('User opens registration form and can see validation errors', () => {
 
     /* The User should be able to see validation errors on files tab */
     cy.get('[data-testid="nav-tabpanel-files-and-license"]').click({ force: true });
-    cy.get('p.Mui-error').should('have.length', 1);
-    cy.get('[data-testid=nav-tabpanel-files-and-license]').within(() =>
-      cy.get('[data-testid=error-tab]').should('exist')
-    );
```

Comment:
Bevare disse testene i stedet for å ta vekk? Om jeg har skjønt riktig er endringen nå at vi _ikke_ ønsker at det skal vises feilmelding i dette tilfellet, så 

```
    cy.get('p.Mui-error').should('have.length', 0); // mulig denne må være should("not.exist") elns
    cy.get('[data-testid=nav-tabpanel-files-and-license]').within(() =>
      cy.get('[data-testid=error-tab]').should('not.exist')
    );
```